consul_config.py.ctmpl

```diff
@@ -41,16 +41,26 @@ POSTGRES_ADMIN_URI = "SERVICEDOESNOTEXIST_pgbouncer-master"
 {{end}}
 
 
-{{if service "pgbouncer-slave"}}
-{{with index (service "pgbouncer-slave") 0}}
+# Use a key '$musicbrainz_up' in consul (json boolean true or false) to decide if we connect to musicbrainz.
+# If we set MB_DATABASE_URI to an empty string, then the @mb_database_needed decorator will prevent the
+# webserver from accessing the musicbrainz database.
+{{- /* This needs to use key/printf again here instead of our pre-defined template at the top of the file as you cannot
+     call a template in an if statement or variable declaration */}}
+{{- $musicbrainz_up := (key (printf "docker-server-configs/LB/config.%s.json/musicbrainz_up" (env "DEPLOY_ENV"))) }}
```

Comment:
not important for this PR, but perhaps we can have another look into how to do this key interpolation, so that we don't have to copy/paste printf statements all over this file...